src/components/Send.jsx

```diff
@@ -92,65 +180,88 @@ export default function Payment({ currentWallet }) {
-      </rb.Form.Group>
-      <rb.Form.Group className="mb-3" controlId="isCoinjoin">
-        <rb.Form.Check
-          type="switch"
-          label="As coinjoin"
-          value={true}
-          onChange={(e) => setIsCoinjoin(e.target.checked)}
-        />
-      </rb.Form.Group>
-      {isCoinjoin === true && (
-        <rb.Form.Group className="mb-3" controlId="counterparties">
-          <rb.Form.Label>Number of counterparties</rb.Form.Label>
-          <rb.Form.Control
-            name="counterparties"
-            type="number"
-            min={0}
-            defaultValue={3}
-            style={{ width: '10ch' }}
-            required
-          />
-          <rb.Form.Control.Feedback type="invalid">Please set the counterparties.</rb.Form.Control.Feedback>
-        </rb.Form.Group>
+    <>
+      {!walletInfo ? (
+        <rb.Row className="justify-content-center">
+          <rb.Col className="flex-grow-0">
+            <div className="d-flex justify-content-center align-items-center">
+              <rb.Spinner as="span" animation="border" size="sm" role="status" aria-hidden="true" className="me-2" />
+              Loading
+            </div>
+          </rb.Col>
+        </rb.Row>
+      ) : (
+        <rb.Row className="send justify-content-center">
+          <rb.Col md={10} lg={8} xl={6}>
+            <PageTitle
+              title="Send bitcoin"
+              subtitle="You can mix your bitcoin before sending them for better privacy." // Todo: Copy.
+            />
+            {alert && <rb.Alert variant={alert.variant}>{alert.message}</rb.Alert>}
+            <rb.Form onSubmit={onSubmit} validated={validated} noValidate>
+              <rb.Form.Group className="mb-4" controlId="destination">
+                <rb.Form.Label>Recipient</rb.Form.Label>
+                <rb.Form.Control
+                  name="destination"
+                  placeholder="Enter address..."
+                  defaultValue=""
+                  className="number"
+                  required
+                />
+                <rb.Form.Control.Feedback type="invalid">Please provide a recipient address.</rb.Form.Control.Feedback>
+              </rb.Form.Group>
+              <rb.Form.Group className="mb-4 flex-grow-1" controlId="account">
+                <rb.Form.Label>Choose account</rb.Form.Label>
+                <rb.Form.Select
+                  defaultValue={account}
+                  onChange={(e) => setAccount(parseInt(e.target.value, 10))}
+                  required
+                  className="number"
+                >
+                  {walletInfo.accounts
+                    .sort((lhs, rhs) => lhs.account - rhs.account)
+                    .map(({ account, account_balance: balance }) => (
+                      <option key={account} value={account}>
+                        Account {account} {settings.showBalance && `(\u20BF${balance})`}
```

Comment:
Balance is always shown as BTC.. which is kind of strange when I have my settings set to `sats` and the next input field wants me to insert a value in sats. 
What do you think about displaying "xxx sats" when `settings.unit` is `SATS`?

Reply:
Let's track that as an improvement if ok for you. We can't use the `<Balance>` component inside an `<option>` unfortunately so we'll probably need some trickery. 🪄

Reply:
Tracked https://github.com/joinmarket-webui/joinmarket-webui/issues/78.